Source/Windows/ARM64EC/Module.cpp

```diff
@@ -364,6 +385,7 @@ static void RethrowGuestException(const EXCEPTION_RECORD& Rec, ARM64_NT_CONTEXT&
   uint64_t GuestSp = Context.X[Config.SRAGPRMapping[static_cast<size_t>(FEXCore::X86State::REG_RSP)]];
   struct DispatchArgs {
     ARM64_NT_CONTEXT Context;
+    uint64_t Pad[4]; // Only present on newer Windows versions, likely for SVE.
```

Comment:
How new are we talking here? Should a version check be somewhere, and handle the different flavours of arguments? Or a version check at startup that early exits without trying? Does WINE versus Windows care about the difference here?

Reply:
I have no idea really, I have only tested on a single windows version (latest insider)  but heard from the wine devs that it wasn't previously present. This is also quite awkward to test at runtime either way. Perhaps it's worth testing on your oryon laptop if you're still running the stock version of windows? I can send something to test

Upstream wine currently doesn't have the padding but I do locally and will PR it - it also doesn't support the exception handling callback that this ends up being called from so it doesn't really end up mattering.

Reply:
I'm not too concerned about it. At least not enough to jump in to Windows for any testing.
If it's on Windows and WINE is going to add the padding to match then it's reasonable enough for me. Even easier if WINE doesn't support hitting this code path yet.